apollo-router/src/plugins/telemetry/tracing/apollo_telemetry.rs

```diff
@@ -229,6 +321,13 @@ impl Exporter {
             errors_configuration: errors_configuration.clone(),
             use_legacy_request_span: use_legacy_request_span.unwrap_or_default(),
             include_span_names: INCLUDE_SPANS.into(),
+            include_attr_names: match apollo_tracing_protocol {
+                // TBD(tim): consider filtering the current implementation for memory purposes to HashSet::from(REPORTS_INCLUDE_ATTRS)
```

Comment:
I think you can remove this comment

Reply:
I was indicating that _today_, router does not filter attributes when storing to the LRU cache for the protobuf reports, however we could consider an optimization for the existing path.  Let me know if you think it's a good idea or we should not modify the "existing" behavior too much in this PR.

Reply:
That sounds like a good idea yes

Reply:
We're doing this now BTW.